src/shared/components/private_message/private-message.tsx

```diff
@@ -276,10 +281,17 @@ export class PrivateMessage extends Component<
           </form>
         )}
         {this.state.showReply && (
-          <PrivateMessageForm
-            recipient={otherPerson}
-            onCreate={this.props.onCreate}
-          />
+          <div className="row">
+            <div className="col-sm-6">
+              <PrivateMessageForm
+                privateMessageView={message_view}
+                replyType={true}
+                recipient={otherPerson}
+                onCreate={this.props.onCreate}
+                onCancel={this.handleReplyCancel}
+              />
+            </div>
+          </div>
```

Comment:
This is an unusual usage of bootstrap's columns. Perhaps you could try giving the form element in the `PrivateMessageForm` component a class of "w-100 w-sm-75 w-lg-50 mx-auto" and seeing how that looks.

Reply:
I don't think those classes exist?: https://github.com/twbs/bootstrap/pull/31323